applications/ultrasound_postprocessing/ultra_post/filters/gamma.py

```
@@ -0,0 +1,30 @@
+
+from __future__ import annotations
+
+import cupy as cp
+
+
+def gamma_compression(image: cp.ndarray, gamma: float = 1.0) -> cp.ndarray:
+    """Power-law gamma correction."""
+
+    gamma = float(gamma)
+    if gamma <= 0.0:
+        raise ValueError("Gamma must be positive.")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)